airflow/providers/common/sql/hooks/sql.py

```diff
@@ -264,6 +266,7 @@ def run(
                 results = []
                 for sql_statement in sql:
                     self._run_command(cur, sql_statement, parameters)
+                    self._update_query_ids(cur)
```

Comment:
At this point, the query is done, so what is the utility of updating the query id list here?

Reply:
You're right. I checked it for Snowflake.
![photo_2022-11-13_00-53-28](https://user-images.githubusercontent.com/10523218/201497699-ed4ed7b1-cd42-4515-acbd-51ae27349094.jpg)

Reply:
Do you have access to any Trino and Presto instances to check it also for these DBs?